- Force-disable udev detection
- Disable building the Python GTK+3 user interface

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -150,11 +150,6 @@ if(CapsImage_FOUND)
 	set(HAVE_CAPSIMAGE 1)
 endif(CapsImage_FOUND)
 
-find_package(Udev)
-if(Udev_FOUND)
-	set(HAVE_UDEV 1)
-endif(Udev_FOUND)
-
 find_package(Capstone)
 if(Capstone_FOUND)
 	set(CMAKE_REQUIRED_INCLUDES ${CAPSTONE_INCLUDE_DIR})
@@ -383,22 +378,6 @@ if(Python_Interpreter_FOUND AND Python_VERSION_MAJOR L
 	message("Note: Hatari needs at least Python 3 ... ignoring older version")
 	unset(Python_Interpreter_FOUND)
 endif()
-set(PYTHON_GTK_FOUND 0)
-if(Python_Interpreter_FOUND)
-	execute_process(COMMAND ${Python_EXECUTABLE} -c "\
-import gi\n\
-gi.require_version('Gtk', '3.0')\n\
-from gi.repository import Gtk\n\
-from gi.repository import Gdk\n\
-from gi.repository import GdkPixbuf\n\
-from gi.repository import GLib"
-			RESULT_VARIABLE PYTHON_GTK_RESULT
-			OUTPUT_QUIET ERROR_QUIET)
-	if(${PYTHON_GTK_RESULT} EQUAL 0)
-		set(PYTHON_GTK_FOUND 1)
-		add_subdirectory(python-ui)
-	endif()
-endif(Python_Interpreter_FOUND)
 
 if(UNIX AND NOT ENABLE_OSX_BUNDLE)
 	add_subdirectory(share)
